src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -511,5 +517,10 @@ public static void EndpointTimeout(this ILogger logger, string processId)
         {
             _endpointTimeout(logger, processId, null);
         }
+
+        public static void LoadingProfiler(this ILogger logger, Guid profilerGuid, string path, int processId)
+        {
+            _loadingProfiler(logger, profilerGuid.ToString(), path, processId.ToString(), null);
```

Comment:
Remove the `ToString` calls and let the format string handle whatever format it wants to represent the arguments (by adding format specifiers to each hole). Otherwise, you should be passing the format specifiers to these `ToString` calls as well as a `CultureInfo` implementation (probably `CultureInfo.InvariantCulture`).